### PR TITLE
Docs update and some minor code fixes

### PR DESCRIPTION
**Pull request by mariuzka**

### CODE REVIEW

**Comment by luerhard** [comment]
Es gibt noch einiges an Fehlern in den Docs:

```
/mnt/nvme_storage/git/pop2net/docs/about.rst:1: WARNING: Title overline too short.

============
About Pop2net
============
/mnt/nvme_storage/git/pop2net/docs/api/magiclocation.rst:4: WARNING: Title underline too short.

MagicLocation
========
/mnt/nvme_storage/git/pop2net/docs/examples/wealth_transfer.ipynb:1: WARNING: Each notebook should have at least one section title
/mnt/nvme_storage/git/pop2net/docs/section_introduction.rst:4: WARNING: toctree contains reference to nonexisting document '#Introduction/introduction_inspector'
looking for now-outdated files... none found
pickling environment... done
checking consistency... /mnt/nvme_storage/git/pop2net/docs/Introduction/introduction_inspector.ipynb: WARNING: document isn't included in any toctree
/mnt/nvme_storage/git/pop2net/docs/api/inspector.rst: WARNING: document isn't included in any toctree
/mnt/nvme_storage/git/pop2net/docs/examples/infection_simulation.ipynb: WARNING: document isn't included in any toctree
/mnt/nvme_storage/git/pop2net/docs/examples/minimal_example.ipynb: WARNING: document isn't included in any toctree
/mnt/nvme_storage/git/pop2net/docs/examples/opinion_dynamics.ipynb: WARNING: document isn't included in any toctree
/mnt/nvme_storage/git/pop2net/docs/examples/real_data.ipynb: WARNING: document isn't included in any toctree
/mnt/nvme_storage/git/pop2net/docs/examples/schelling.ipynb: WARNING: document isn't included in any toctree
/mnt/nvme_storage/git/pop2net/docs/examples/wealth_transfer.ipynb: WARNING: document isn't included in any toctree
done

```

**Comment by mariuzka** [comment]
<p>Ah da hab ich was bezüglich der Reisekostenabrechnung berechnet xD</p>

---

 <a href='https://app.reviewnb.com/mariuzka/pop2net/pull/97/discussion/#comment-2374624881'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by mariuzka** [comment]
<p>good idea</p>

---

 <a href='https://app.reviewnb.com/mariuzka/pop2net/pull/97/discussion/#comment-2374624903'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

File: tests/test_creator/test_creator_get_affiliated_agents.py [hunk 1]
```diff
@@ -23,6 +23,17 @@ class School(p2n.MagicLocation):
         for agent in creator._get_affiliated_agents(agents=model.agents, dummy_location=school)
     )
 
+    model = p2n.Model()
+    creator = p2n.Creator(model=model)
+
+    for _ in range(5):
+        agent = p2n.Agent(model=model)
+        agent.gender = "w"
```
Comment (luerhard) [inline tests/test_creator/test_creator_get_affiliated_agents.py:31]: I still really hate this syntax xD -- this does not convey to me that an agent is added to the model. model.add_agent(agent) is so much clearer :P

**Comment by mariuzka** [comment]
I will address the points raised tomorrow. :)

**Review by luerhard — APPROVED** [review]

**Pull request merged**